micro-demos/services/demo.sh

```diff
@@ -15,18 +15,18 @@
 
 . $(dirname ${BASH_SOURCE})/../util.sh
 
-if kubectl --namespace=demos get rc hostnames >/dev/null 2>&1; then
-    desc "Revisit our replication controller"
-    run "kubectl --namespace=demos get rc hostnames"
+if kubectl --namespace=demos get rs hostnames >/dev/null 2>&1; then
+    desc "Revisit our replica set"
+    run "kubectl --namespace=demos get rs hostnames"
 else
-    desc "Run some pods under a replication controller"
-    run "kubectl --namespace=demos run hostnames \\
```

Comment:
It's fine to have your RS demo use yaml, but let's leave this one using run and expose, and just update for "deployment".